<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cmdsettings</class>
 <widget class="QDialog" name="cmdsettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="cmdlist">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>190</width>
     <height>290</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>190</width>
     <height>290</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>190</width>
     <height>290</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>5</y>
     <width>190</width>
     <height>290</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>190</width>
     <height>290</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>190</width>
     <height>290</height>
    </size>
   </property>
   <property name="title">
    <string>详情</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>172</width>
      <height>227</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_name">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>50</height>
        </size>
       </property>
       <property name="title">
        <string>名称</string>
       </property>
       <widget class="QLineEdit" name="lineEdit_name">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>150</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_name_2">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>50</height>
        </size>
       </property>
       <property name="title">
        <string>命令</string>
       </property>
       <widget class="QLineEdit" name="lineEdit_cmd">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>150</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_flag">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>80</height>
        </size>
       </property>
       <property name="title">
        <string>标志</string>
       </property>
       <widget class="QCheckBox" name="checkBox_1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>遍历输入文件夹之前</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>遍历输入文件夹之后</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>构建完成</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_new">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>新建</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_save">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_delete">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_help">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>帮助</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
